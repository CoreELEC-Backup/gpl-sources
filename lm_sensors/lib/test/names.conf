# keyword with some unquoted names
label foo bar

# keyword with escaped newline, names, and whitespace
label		\
 foo bar

# names with full range of alpha-numeric and underscores
label \
	abcdefg \
	hijklmnop \
	qrs \
	tuv \
	wx \
	yz \
	a0123456789 \
	982lksdf \
	_abcd \
	1234_ \
	_ \
	foo_bar_baz \
	liajesiajef82197fjadf

# keyword with names, immediate EOF
label foo bar