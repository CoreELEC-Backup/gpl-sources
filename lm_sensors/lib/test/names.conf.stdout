2: LABEL
2: NAME: foo
2: NAME: bar
3: EOL
5: LABEL
6: NAME: foo
6: NAME: bar
7: EOL
9: LABEL
10: NAME: abcdefg
11: NAME: hijklmnop
12: NAME: qrs
13: NAME: tuv
14: NAME: wx
15: NAME: yz
16: NAME: a0123456789
17: NAME: 982lksdf
18: NAME: _abcd
19: NAME: 1234_
20: NAME: _
21: NAME: foo_bar_baz
22: NAME: liajesiajef82197fjadf
23: EOL
25: LABEL
25: NAME: foo
25: NAME: bar
25: EOL
25: EOF
